docs/components/buttons-and-links/button.md

```diff
@@ -24,7 +24,14 @@ There are several types of Buttons with varying degree of emphasis. Emphasis det
 
 ## Sizes
 
-![Button size](/img/buttons/button-size.png)
+![Button size](../../assets/button-size.png)
```

Comment:
I am not sure when building the site, webpack will load the images. I will try it out.

Reply:
understood

Reply:
I pulled your feature branch and tested, I think it works fine. you can move the related images from img folder to here